bucket_39/libreoffice/specification

```diff
@@ -101,7 +101,7 @@ BUILDRUN_DEPENDS=	serf:single:standard
 			mythes:single:standard
 			poppler:cpp:standard
 			lp_solve:single:standard
-			glew:primary:standard
```

Comment:
this is incorrect.
You would leave this line, but put "glew:dev:standard" in the BUILD_DEPENDS section.